app/javascript/react/screens/App/Overview/OverviewConstants.js

```diff
@@ -10,6 +10,10 @@ export const CONTINUE_TO_PLAN = 'CONTINUE_TO_PLAN';
 export const V2V_SET_MIGRATIONS_FILTER = 'V2V_SET_MIGRATIONS_FILTER';
 export const V2V_RETRY_MIGRATION = 'V2V_RETRY_MIGRATION';
 
+export const FETCH_PROVIDERS = 'FETCH_PROVIDERS';
+export const FETCH_PROVIDERS_URL =
+  '/api/providers?expand=resources&attributes=type&filter[]=or%20type=ManageIQ::Providers::Redhat::InfraManager&filter[]=type=ManageIQ::Providers::Vmware::InfraManager';
```

Comment:
are we sure we want to filter providers by these two types? Just double checking ;)

It seems like for now this is the way to go...

Reply:
That's what I got from https://github.com/ManageIQ/miq_v2v_ui_plugin/issues/163 and slack discussions. Right now the only migration path we have is VMware <--> RHV, so those two specific providers are required. If you remove the `&filter` part of this URL, the rest of the code will work fine, it'll just be looping over a larger array when it checks for those types. So we can make this more generic if we need more provider data later.

Reply:
@priley86, also, i should point out that this filter URL came directly from @AparnaKarve, so that's why I did it this way.

Reply:
The above API gives us an output that is focussed on a very specific section of the `providers` API - the `type` of the provider. Also it filters out only the RHV and VMWare providers.

Like @mturley said, we can always modify this API to give us a more generic Provider API output (which will result in a larger json), but for now the above works.

Reply:
great! thanks.